composer.json

```diff
@@ -88,7 +88,7 @@
         "friendsofphp/php-cs-fixer": "~2.14.0",
         "lusitanian/oauth": "~0.8.10",
         "magento/magento-coding-standard": "*",
-        "magento/magento2-functional-testing-framework": "2.5.4",
+        "magento/magento2-functional-testing-framework": "^2.6.0",
```

Comment:
^ does not match the title of this PR

^ would allow upgrades up til 3.0.0
~2.6.0 would allow all 2.6.* versions

Reply:
You are right. Thanks for noticing that fact.